src/main/java/com/marek/onlinebookstore/security/token/AuthenticationService.java

```diff
@@ -0,0 +1,28 @@
+package com.marek.onlinebookstore.security.token;
+
+import com.marek.onlinebookstore.dto.user.login.UserLoginRequestDto;
+import com.marek.onlinebookstore.dto.user.login.UserLoginResponseDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class AuthenticationService {
+    private final JwtUtil jwtUtil;
+    private final AuthenticationManager authenticationManager;
+
+    public UserLoginResponseDto authenticate(UserLoginRequestDto requestDto) {
+        Authentication authenticate = authenticationManager.authenticate(
+                new UsernamePasswordAuthenticationToken(
```

Comment:
Consider it with wrapping this call with `try-catch` block as for example we can handle things like `BadCredentialsException` and then provide appropriate message to the user.